packages/app/src/components/CardWithAttributes.tsx

```diff
@@ -97,7 +105,9 @@ export function CardWithAttributes({
             </Heading>
           </YStack>
           <XStack h="$3">
-            {issuerImage?.url && <Image src={issuerImage.url} alt={issuerImage.altText} width={36} height={36} />}
+            {issuerImage?.url && !isRevokedOrExpired && (
```

Comment:
Why don't we show the issuer image if revoked or expired?

Reply:
You still recognize the credential by it's background so it's not that big of a deal.